roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -29,7 +29,7 @@ l_openshift_prometheus_serviceaccount_annotations:
 openshift_prometheus_additional_rules_file: null
 
 #prometheus application arguments
-openshift_prometheus_args: ['--storage.tsdb.retention=6h']
+openshift_prometheus_args: ['--storage.tsdb.retention=3d']
```

Comment:
I don't see that change in the template or in openshift/origin#18391. Is it in a different open PR in origin?

Reply:
We need a separate PR there.